### ISSUE TITLE
`npx hardhat` arguments are incorrectly parsed by `turbo`

### ISSUE THREAD

**Issue by alcuadrado** [stackblitz/webcontainer-core#277]
### Describe the bug

I'm trying to use [`hardhat`](https://hardhat.org) with stackblitz (great progress btw!), and I get an error when I run `npx hardhat test --show-stack-traces`.

`--show-stack-traces` gets parsed by `turbo`, and I get this error:

```
Unknown Syntax Error: Command not found; did you mean one of:

  0. turbo exec [-p,--package #0] [-c,--call #0] [--cwd,--prefix #0] [-s,--silent] [--quiet] [-V,--verbose] [--loglevel #0] ...
  1. turbo exec [-p,--package #0] [-c,--call #0] [--cwd,--prefix #0] [-s,--silent] [--quiet] [-V,--verbose] [--loglevel #0] ...
  2. turbo exec [-p,--package #0] [-c,--call #0] [--cwd,--prefix #0] [-s,--silent] [--quiet] [-V,--verbose] [--loglevel #0] ...

While running exec hardhat test --show-stack-traces
```

I have no idea what `turbo` is, but I guess it's the console? 

`hardhat test --show-stack-traces` works.

### Link to the blitz that caused the error

https://stackblitz.com/edit/node-is5aai

### Steps to reproduce

Run `npx hardhat test --show-stack-traces` after installing the project

### Expected behavior

It should run `hardhat`

### Platform

```
Browser name  = Chrome 
Full version  = 92.0.4515.115
Major version = 92
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.115 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 80042020,
  "usedJSHeapSize": 72803068,
  "jsHeapSizeLimit": 4294705152
}
Hash = 6a754de1dca8799163c05b54a4efbad0569024b8
```

**Comment by jrvidal** [comment]
Hello @alcuadrado, FYI turbo is our [custom NPM client](https://developer.stackblitz.com/docs/platform/turbo/). This is probably a compat bug on our part, thanks for reporting.

**Comment by sulco** [comment]
For what it's worth, until it's fixed you can use an additional double-hyphen (`--`) to unblock your workflow @alcuadrado.

So instead of:
`npx hardhat test --show-stack-traces`

you can do:
`npx hardhat test -- --show-stack-traces`

**Comment by d3lm** [comment]
This has been fixed. If you run into any more issues, please open a new issue.
